src/main/resources/application-test-file.properties

```diff
@@ -0,0 +1,4 @@
+server.port=8081
+database.type=in-file
+database.path=test_db/invoices.json
+database.idpath=test_db/nextId.txt
```

Comment:
Po obejrzeniu wykładu zauważyłem, że tu jednak jest błąd. Patryk mówił, że propertiesy testowe powinny się znajdować w resources w katalogu test. Jak włączysz wykład to jest o tym mowa 1:28:40.

Reply:
Zrobiłem kopię application.properties w src/test/resources folder i przesunąłem tam application-test.properties.
Nowy PR do sprawdzenia.